helm_deploy/cla-backend/values-dev.yaml

```diff
@@ -16,5 +16,46 @@ service:
 
 # Lists don't deep merge, so this list of envVars overrides anything defined in an earlier values file
 envVars:
-  - name: DEBUG
-    value: "True"
\ No newline at end of file
+  DEBUG:
+    value: "True"
+  DB_HOST:
+    secret:
+      optional: true
+  DB_NAME:
+    secret:
+      optional: true
+  DB_USER:
+    secret:
+      optional: true
+  DB_PASSWORD:
+    secret:
+      optional: true
+  LOAD_TEST_DATA:
+    value: "True"
+  ADMIN_USER:
+    value: "admin"
+    secret: null
```

Comment:
What are the `secret: null` entries for? Are they to override the defaults from `values.yaml`, and it merges objects so you have to explicitly nullify keys rather than just omitting them?

Reply:
Yes it's to override the defaults from values.yaml and set development values for local development instead of reading them from secrets.

>so you have to explicitly nullify keys rather than just omitting them?
Yes

Reply:
i think it only overrides the sub keys that you specify and not the the entire ADMIN_USER object